### PR TITLE
[platform-device/celestica]: add seastone2 platform and device

### PR DESCRIPTION
**Pull request by kantaphon**
**- What I did**
- Port device and platform code of seastone2 master branch to 201811 branch for azure.

**- How I did it**

- Add and modify code to support seastone2 device.
- Add and modify code to support seastone2 platform.

**- How to verify it**

- Check if the changes pass compilation.
- Run platform api test on testbed.

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: platform/broadcom/one-image.mk [hunk 1]
```diff
@@ -34,7 +34,8 @@ $(SONIC_ONE_IMAGE)_LAZY_INSTALLS += $(DELL_S6000_PLATFORM_MODULE) \
                                $(QUANTA_IX1B_32X_PLATFORM_MODULE) \
                                $(MITAC_LY1200_32X_PLATFORM_MODULE) \
                                $(ALPHANETWORKS_SNH60A0_320FV2_PLATFORM_MODULE) \
-                               $(ALPHANETWORKS_SNH60B0_640F_PLATFORM_MODULE)
+                               $(ALPHANETWORKS_SNH60B0_640F_PLATFORM_MODULE) \
+                               $(CEL_SEASTONE2_PLATFORM_MODULE)
```
Comment (mudsut4ke) [inline platform/broadcom/one-image.mk:38]: Please move '$(CEL_SEASTONE2_PLATFORM_MODULE)' to line number 29 to join with other celestica devices

**Comment by mudsut4ke** [comment]
@kantaphon  , Please also add this folder 
https://github.com/Azure/sonic-buildimage/tree/master/platform/broadcom/sonic-platform-modules-cel/services/platform_api

**Comment by kantaphon** [comment]
> @kantaphon , Please also add this folder
> https://github.com/Azure/sonic-buildimage/tree/master/platform/broadcom/sonic-platform-modules-cel/services/platform_api

Please help to review the last commit ID: [https://github.com/SONIC-DEV/sonic-buildimage/pull/242/commits/61a080f775efbc93480f04e97bd97c19cb5bce17](url)

**Comment by mudsut4ke** [comment]
Please also update rule for seastone2 follow this design but please ignore python3 command
https://github.com/Azure/sonic-buildimage/blob/master/platform/broadcom/sonic-platform-modules-cel/debian/rules

**Pull request merged**